app/rne-theme/colors.ts

```diff
@@ -1,12 +1,14 @@
-const colors = {
+import { palette } from "@app/theme"
+
+const light = {
   primary: "#1627C4",
-  primary3: "#1627C4",
```

Comment:
What was the thinking behind changing these colors? The naming came from our Figma design and existing components already depend on these names.

Reply:
I see in a separate commit you updated the existing components. There is still a cognitive overhead because all of the components in Figma use this naming.

Reply:
the idea is to have them in the right order in storybook

![image](https://user-images.githubusercontent.com/11156370/227750992-6afe55c2-790b-4cfd-9a11-c544654025cd.png)

didn't know the naming was coming from Figma. I can revert it.

Reply:
I think it would be ideal to revert it. If we want it sorted we can manually specify the order in the component.